GitUI/UserControls/RevisionGrid/FilterInfo.cs

```diff
@@ -240,55 +240,68 @@ public bool Apply(RevisionFilter filter)
                 Debug.Fail("Not supported");
             }
 
+            var searchParametersChanged = filter.FilterByAuthor != ByAuthor
```

Comment:
Please don't use `var` for intrinsic types. Also don't use when the type isn't obvious (e.g., `var result = DoWork();`).

Reply:
I tend to think that boolean is a type where it is obvious enough but ok.